client/reader/search-stream/index.jsx

```diff
@@ -206,6 +206,30 @@ const SearchStream = React.createClass( {
 		return SearchCardAdapter( isRecommendations );
 	},
 
+	handleStreamMounted( c ) {
+		this.streamRef = c;
+	},
+
+	handleSearchBoxMounted( c ) {
+		this.searchBoxRef = c;
+	},
+
+	resizeSearchBox() {
+		if ( this.searchBoxRef && this.streamRef ) {
+			const width = this.streamRef.getClientRects()[ 0 ].width;
+			this.searchBoxRef.style.width = `${ width }px`;
```

Comment:
👍 

How about if `getClientRects()` didn't exist on the ref? In a unit test for example:

https://github.com/jquery/jquery/issues/2804